specification/dartLangSpec.tex

```diff
@@ -10746,9 +10858,12 @@ \subsubsection{Compound Assignment}
 Consider a compound assignment $a$ of the form \code{$e_1$.$v$ ??= $e_2$}.
 Let $T$ be the static type of $e_1$ and let $x$ be a fresh variable of type $T$.
 Except for errors inside $e_1$ and references to the name $x$,
-exactly the same compile-time errors that would be caused by \code{$x$.$v$ = $e_2$} are also generated in the case of $a$.
+exactly the same compile-time errors that would be caused by
+\code{$x$.$v$ = $e_2$}
+are also generated in the case of $a$.
 %% TODO(eernst): Also, we should mention other cases, e.g., there is no getter `z.v`.
```

Comment:
That must be a renaming glitch. It should say `x.v`. The point is that the dynamic semantics will evaluate `x.v`, so we should have a compile-time error when there is a setter but no getter in the interface of the type of `x`. Adjusted the text to say this, rather than having a TODO that might as well just be resolved here.

Made the same change to the following two cases (lines 10885, 10909).